examples/variational_autoencoder.py

```diff
@@ -20,15 +20,15 @@
 epsilon_std = 1.0
 
 
-x = Input(batch_shape=(batch_size, original_dim))
+x = Input(shape=(original_dim,))
 h = Dense(intermediate_dim, activation='relu')(x)
 z_mean = Dense(latent_dim)(h)
 z_log_var = Dense(latent_dim)(h)
 
 
 def sampling(args):
     z_mean, z_log_var = args
-    epsilon = K.random_normal(shape=(batch_size, latent_dim), mean=0.,
+    epsilon = K.random_normal(shape=(K.shape(z_mean)[0], latent_dim), mean=0.,
```

Comment:
Good to know, it seems this is something that was fixed recently on the TF side. For a long time it was impossible to construct new symbolic shapes by mixing entries from existing symbolic shapes and Python integers.